doc/manual/src/SUMMARY.md.in

```diff
@@ -38,7 +38,7 @@
     - [Arguments and Variables](expressions/arguments-variables.md)
     - [Building and Testing](expressions/simple-building-testing.md)
     - [Generic Builder Syntax](expressions/generic-builder.md)
-  - [Writing Nix Expressions](expressions/expression-language.md)
+  - [Nix Expression Language](expressions/expression-language.md)
```

Comment:
Oh good catch! "Writing Nix Expressions" is a few lines above. Want to PR this to `master` right away?

Reply:
I don't care, eventually it will end up there. :)